TestProject_Calc/CalculatorPage.cs

```diff
@@ -1,7 +1,184 @@
-﻿namespace TestProject_Calc
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Support.UI;
+using System.Reflection;
+
+namespace TestProject_Calc
 {
-    public class CalculatorPage()
+    public class CalculatorPage(ChromeDriver driver)
     {
         public static string URL = "http://localhost:5000/Calculator";
+        private IWebElement DepositAmmountField => driver.FindElement(By.XPath("//input[@id='amount']"));
```

Comment:
XPath is ok, you can locate any element with it. But don't forget about By.Id, By.Class, etc.
```suggestion
        private IWebElement DepositAmmountField => driver.FindElement(By.Id("amount"));
```